src/mumble/DirectSound.cpp

```diff
@@ -560,15 +560,15 @@ void DXAudioInput::run() {
 					didsleep = true;
 					firstsleep = false;
 				}
-			} while (static_cast<int>(dwReadyBytes) < sizeof(short) * iFrameSize);
+			} while (static_cast<size_t>(dwReadyBytes) < sizeof(short) * iFrameSize);
 
 			// Desynchonized?
 			if (dwReadyBytes > (dwBufferSize / 2)) {
 				qWarning("DXAudioInput: Lost synchronization");
 				dwLastReadPos = dwReadPosition;
 			} else if (bRunning) {
 				if (FAILED(hr = pDSCaptureBuffer->Lock(dwLastReadPos, sizeof(short) * iFrameSize, &aptr1, &nbytes1, &aptr2, &nbytes2, 0))) {
-					qWarning("DXAudioInput: Lock from %ld (%d bytes) failed: hr=0x%08lx",dwLastReadPos, sizeof(short) * iFrameSize, hr);
+					qWarning("DXAudioInput: Lock from %lu (%zu bytes) failed: hr=0x%08lx", static_cast(dwLastReadPos), sizeof(short) * iFrameSize, hr);
```

Comment:
This doesn't build...

Should be `static_cast<unsigned long>(...)`... Apologies if I wrote only `static_cast(...)` before. The markdown must have eaten my `<` and `>`s.